manipulation/src/main/java/org/jboss/gm/manipulation/ManipulationPlugin.java

```diff
@@ -27,11 +25,14 @@
 import org.jboss.gm.manipulation.actions.UploadTaskTransformerAction;
 
 import static org.apache.commons.lang.StringUtils.isEmpty;
+import static org.apache.commons.lang.StringUtils.isNotEmpty;
 
 @SuppressWarnings("unused")
```

Comment:
Looks like we should also remove this suppression. The only thing it hides is the non-used isDependencyManagementPluginPomCustomizationEnabled method :).

Reply:
Good catch!